packages/client/src/v2-events/features/events/registered-fields/AdministrativeArea.tsx

```diff
@@ -72,7 +72,7 @@ function AdministrativeAreaOutput({ value }: { value: Stringifiable }) {
 
 function useStringifier() {
   const locations = useSelector(getLocations)
-  return (value: string) => locations[value].name
+  return (value: string) => (value ? locations[value].name : undefined)
```

Comment:
We could pass an empty string for empty values, that i think prevents us from using this `{child.birthLocation, select, undefined {} other {{child.birthLocation}}}{child.address.district, select, undefined {} other {, {child.address.district}}}{child.address.province, select, undefined {} other {, {child.address.province}}}`

We cannot really detect for an empty string in a message, can we?

Reply:
@Nil20 we are just now introducing a `__EMPTY__` placeholder that could be used here? Might already be in develop

Reply:
Yeah we can. Updated